scripts/spack/configs/blueos_3_ppc64le_ib_p9/packages.yaml

```diff
@@ -41,16 +41,6 @@ packages:
     variants: ~dbm
```

Comment:
This is just updating this file to be inline with the devtools file and alphabetizing